usr/src/boot/i386/libi386/biosdisk.c

```diff
@@ -1043,7 +1043,7 @@ bd_realstrategy(void *devdata, int rw, daddr_t dblk, size_t size,
 	}
 	if (bbuf == NULL) {
 		bio_size = V86_IO_BUFFER_SIZE;
-		if (bio_size / bd->bd_sectorsize == 0)
```

Comment:
hmmm ... how did that ever pass muster

Reply:
No idea! It came across from FreeBSD and was committed over there back in 2018.